sql_generators/active_users/templates/desktop_query.sql

```diff
@@ -74,12 +74,12 @@ SELECT
     active_hours_sum,
     first_seen_date
   ),
-  COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS dau,
+  COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS daily_users,
   COUNT(DISTINCT IF(days_since_seen < 7, client_id, NULL)) AS wau,
   COUNT(DISTINCT client_id) AS mau,
```

Comment:
Any thoughts on the fact that what we call WAU and MAU is no longer consistent with DAU? @lucia-vargas-a @jmsilverman @jdma-mozilla 

Should we address that at some point? I currently have a [request](https://mozilla-hub.atlassian.net/browse/DS-3252) to grab WAU and MAU numbers for mobile and it seems the right thing to do would be to filter for duration, in particular for iOS.

Reply:
Thanks for the context! I think to be consistent we should change the names of `mau` and `wau` here to `monthly_users` and `weekly_users`, respectively, *but we should also* make sure that the names surfaced outside of this file don't change (i.e., we surface them elsewhere as `dau`, `wau`, and `mau`).
I figured the lack of a '28 day filter' thing was due to where it was getting the data from.

Reply:
Yes, the upstream table captures [activity over 28 days](https://github.com/mozilla/bigquery-etl/blob/5cccb92a342c9fd43f713d77665f668d6c158e79/sql/moz-fx-data-shared-prod/org_mozilla_firefox_derived/baseline_clients_last_seen_v1/query.sql#L4).

Reply:
I think the re-naming makes sense! If we agree to do so I'll raise a separate small PR.

Reply:
DRAFT-PR https://github.com/mozilla/bigquery-etl/pull/4550